lib/matplotlib/axes/_axes.py

```diff
@@ -5287,12 +5275,27 @@ def imshow(self, X, cmap=None, norm=None, aspect=None,
             The Colormap instance or registered colormap name used to map
             scalar data to colors. This parameter is ignored for RGB(A) data.
 
-        norm : `~matplotlib.colors.Normalize`, optional
+        norm : str or `~matplotlib.colors.Normalize`, optional
             The `.Normalize` instance used to scale scalar data to the [0, 1]
```

Comment:
Let's keep this general, i.e. type agnostic.
```suggestion
            The normalization method used to scale scalar data to the [0, 1]
```